qaevents.md

```diff
@@ -0,0 +1,116 @@
+  "status": "PENDING",
+  "message" : {
+    "type":"doi",
+    "value":"10.18848/1447-9494/cgp/v15i09/45934",
+    "abstracts":"Abstract Complex functionality of the ...",
+    "acronym":"EPSILON",
+    "code":"277749",
+    "funder":"EC",
+    "fundingProgram":"FP7",
+    "jurisdiction":"EU",
+    "title":"Elliptic Pdes and Symmetry of Interfaces and Layers for Odd Nonlinearities"
+  },
+  type: "qaevent"
+}
+```
+
+Attributes
+* the *id* attribute is the event primary key
+* the *originalId* attribute is the oai identifier of the target publication
+* the *title* attribute is the title of the publication as provided by the correction's source
+* the *trust* attribute is the level of accuracy of the suggestion
+* the *status* attribute is one of (ACCEPTED, REJECTED, DISCARDED, PENDING)
+* the *eventDate* attribute is the timestamp of the event reception
+* the *message* attribute is a json object which structure depends on the source and on the topic of the event. When the "topic" type is
+    * ENRICH/MISSING/PID and ENRICH/MORE/PID: fills `message.type` with the type of persistent identifier (doi, pmid, etc.) and `message.value` with the corresponding value
+    * ENRICH/MISSING/ABSTRACT: fills `message.abstract`
+    * ENRICH/MISSING/SUBJECT/ACM: fills the `message.value` with the actual keywords, the subject classification is defined by the last part of the topic (ACM, JEL, DDC, etc.)
+    * ENRICH/MISSING/PROJECT: fills `acronym`, `code`, `funder`, `fundingProgram`, `jurisdiction` and `title`
+
+Exposed links:
+* topic: link to the topic to which the event belong to
+* target: link to the item that represent the targe to whom the suggestion apply
+* related: link to a second item that is involved in the qa events (i.e. the project item)
+
+Status codes:
+* 200 Ok - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged as an administrator
+* 404 Not found - if no qa event exists with such id 
+
+## Search methods
+### Get qaevents by a given topic
+**/api/integration/qaevents/search/findByTopic?topic=:target-key[&size=10&page=0]**
+
+It returns the list of qa events from a specific topic
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* topic: mandatory, the key associated with the requested topic
+
+Return codes:
+* 200 OK - if the operation succeed
+* 400 Bad Request - if the topic parameter is missing or invalid
+
+Provide paginated list of the qa events available.
+
+## PATCH 
+### To record a decision 
+PATCH /api/integration/qaevents/<:qaevent-id>
+
+This method allow users to Accept, Reject or Discard a qa event. The PATCH body must follow the JSON PATCH specification
+
+```json
+[
+  {
+    "op": "replace",
+    "path": "/status",
+    "value": "ACCEPTED|REJECTED|DISCARDED|PENDING"
+  }
+]
+```
+
+As response, the modified qa event will be returned.
+ 
+## POST
+### To associated a related item to the qa event
```

Comment:
As implied above, I think we need more details about "related items".  Are they always Project Items?  Are they required?  When or why would you need to "associate a related item"?